.github/ISSUE_TEMPLATE/exploratory-charter.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Exploratory Charter
+about: Suggest a structured exploration
+title: '[Explore] <title>'
+labels: research
+assignees: ''
+
+---
+
+## Charter
+
+**Explore** [target]
+**With** [resources]
+**To discover** [information]
```

Comment:
Would a link to the blog post describing this format be useful here? I know that helped me a lot on another project where we were using [Conventional Comments](https://conventionalcomments.org/) to remember exactly what they meant each time I did a review.

Reply:
Yes, definitely! Will add.